src/ui/screens/AddBountyScreen.tsx

```diff
@@ -0,0 +1,160 @@
+    if (account && api && bountyAllocationBN) {
+      startTx({
+        address: account.address,
+        txMethod: `${api.tx.bounties ? `bounties.proposeBounty` : `treasury.proposeBounty`}`,
+        params: [bountyAllocationBN, bountyTitle],
+      })
+        .then(() => {
+          snackbar('New bounty created');
+        })
+        .catch(() => {
+          snackbar('Error while submitting bounty');
+        });
+      modalRef.current?.close();
+    }
+  };
+
+  return (
+    <Modalize ref={modalRef} adjustToContentHeight onClose={navigation.goBack} panGestureEnabled={false}>
+      <Layout style={styles.container}>
+        <View style={styles.header}>
+          <Headline>{'Add Bounty'}</Headline>
+        </View>
+        <View style={styles.container}>
+          <InputLabel label={'Bounty Title:'} helperText={'Description of the Bounty (to be stored on-chain)'} />
+          <TextInput
+            autoFocus
+            placeholder="Enter bounty title"
+            value={bountyTitle}
+            onChangeText={(value) => setBountyTitle(value)}
+          />
+          {isBountyTitleValid && bountyTitle.length !== 0 ? (
+            <Text style={{color: colors.error}}>Exceeding maximum reasoning length</Text>
+          ) : null}
+          <Padder scale={1} />
+          <InputLabel
+            label={'Bounty Requested Allocation:'}
+            helperText={
+              'How much should be paid out for completed Bounty. Upon funding, the amount will be reserved in treasury.'
+            }
+          />
+          <TextInput
+            keyboardType="decimal-pad"
+            placeholder="Enter bounty requested allocation"
+            value={bountyAllocation}
+            onChangeText={(value) => {
+              SetBountyAllocation(decimalKeypad(value));
+            }}
+            contextMenuHidden={true}
+            right={<TextInput.Affix text={formatBalance(stringToBn(bountyAllocation)) ?? ''} />}
+          />
+          {isBountyValid ? <Text>Minimum bounty reward is: {formatBalance(bounty?.bountyValueMinimum)}</Text> : null}
+          <Padder scale={1} />
+          <InputLabel label={'Bounty Bond:'} helperText={'Proposer bond depends on bounty title length.'} />
+          <TextInput placeholder="Bounty Bond" value={formatBalance(calculatedBountyBond)} disabled />
+          <Padder scale={1} />
+          <InputLabel
+            label={'Submit with account'}
+            helperText={'This account will propose the bounty. Bond amount will be reserved on its balance.'}
+          />
+          <SelectAccount onSelect={(selectedAccount) => setAccount(selectedAccount.accountInfo)} />
+          <Padder scale={1} />
+          <View style={styles.row}>
+            <Button mode="outlined" onPress={modelClose}>
+              Cancel
+            </Button>
+            <Button mode="outlined" disabled={disabled} onPress={submitBounty}>
+              Submit Bounty
+            </Button>
+          </View>
```

Comment:
Apply a bottom `<Padder scale={1} />` here so that the buttons don't go to the bottom edge.